packages/mermaid/src/rendering-util/rendering-elements/shapes/util.ts

```diff
@@ -68,7 +77,11 @@ export const labelHelper = async (parent, node, _classes) => {
                     ? getConfig().fontSize
                     : window.getComputedStyle(document.body).fontSize;
                   const enlargingFactor = 5;
-                  const width = parseInt(bodyFontSize, 10) * enlargingFactor + 'px';
+                  const parsedBodyFontSize =
+                    typeof bodyFontSize === 'number'
+                      ? bodyFontSize
+                      : parseInt(bodyFontSize ?? '', 10);
```

Comment:
We can use this.

https://github.com/mermaid-js/mermaid/blob/2933eb5c288784650d0499a878f98fff8c9790e4/packages/mermaid/src/utils.ts#L832-L856

Reply:
Ah, I didn't know that function existed!

Fixed by https://github.com/mermaid-js/mermaid/pull/5974/commits/bb97fdbd1e539f9585a613844b06b6593d09e44d

I also added a fallback to `defaultConfig.fontSize` if the value is a `NaN`.